go-controller/pkg/types/const.go

```diff
@@ -32,6 +32,10 @@ const (
 
 	NodeLocalSwitch = "node_local_switch"
 
+	// ACL directions
+	DirectionToLPort   = "to-lport"
```

Comment:
I don't really expect people to migrate to using this constant. I would rather seem them migrate to go-ovn bindings.

Reply:
Not sure what you are referring to? This line is untouched by this commit

Reply:
The commit shows you added this line:
```
	DirectionToLPort   = "to-lport"
```

Reply:
Ah, sorry. Must have been my github review layout which confused me. 

> I would rather seem them migrate to go-ovn bindings.

So you want me to remove this commit? The reason why I introduced it is to avoid having cyclic dependencies between the packages consuming this constant.

Reply:
no, you dont have to remove it. I was just stating that it will probably go away when we move to bindings.